cypress/e2e/orders/renewal-field-prefilled-from-template.cy.js

```diff
@@ -0,0 +1,84 @@
+import { DevTeams, TestTypes, Permissions, Parallelization } from '../../support/dictionary';
+import { Orders, OrderLineDetails, OrderDetails } from '../../support/fragments/orders';
+import { NewOrganization, Organizations } from '../../support/fragments/organizations';
+import { OpenOrder, OrderTemplates } from '../../support/fragments/settings/orders';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import { ACQUISITION_METHOD_NAMES_IN_PROFILE, ORDER_FORMAT_NAMES } from '../../support/constants';
+
+describe('Orders', () => {
+  const isOpenOrderEnabled = true;
+  const testData = {
+    organization: NewOrganization.getDefaultOrganization(),
+    orderTemplate: OrderTemplates.getDefaultOrderTemplate({ orderType: 'Ongoing' }),
+    user: {},
+  };
+
+  before('Create test data', () => {
+    cy.getAdminToken().then(() => {
+      OpenOrder.setOpenOrderValue(isOpenOrderEnabled);
+      OrderTemplates.createOrderTemplateViaApi(testData.orderTemplate);
+      Organizations.createOrganizationViaApi(testData.organization);
+    });
+
+    cy.createTempUser([
+      Permissions.uiOrdersCreate.gui,
+      Permissions.uiOrdersEdit.gui,
+      Permissions.uiOrdersView.gui,
+    ]).then((userProperties) => {
+      testData.user = userProperties;
+
+      cy.login(testData.user.username, testData.user.password, {
+        path: TopMenu.ordersPath,
+        waiter: Orders.waitLoading,
+      });
+    });
+  });
+
+  after('Delete test data', () => {
+    cy.getAdminToken().then(() => {
+      OpenOrder.setOpenOrderValue(false);
+      Organizations.deleteOrganizationViaApi(testData.organization.id);
+      OrderTemplates.deleteOrderTemplateViaApi(testData.orderTemplate.id);
+      Users.deleteViaApi(testData.user.userId);
+    });
+  });
+
+  it(
+    'C353575 Renewal note field is prefilled in POL when create order from specific template (thunderjet) (TaaS)',
+    { tags: [TestTypes.extendedPath, DevTeams.thunderjet, Parallelization.nonParallel] },
+    () => {
+      // Click "Actions" button on "Orders" pane and select "New" option
+      const OrderEditForm = Orders.clickCreateNewOrder();
+
+      // Select template name from Precondition in "Template name" dropdown
+      OrderEditForm.selectOrderTemplate(testData.orderTemplate.templateName);
+
+      // Fill all required fields with valid values and click "Save & close" button
+      OrderEditForm.selectVendorByName(testData.organization.name);
+      OrderEditForm.clickSaveButton();
+
+      // Click "Actions" button, Select "Add PO line" option
+      const OrderLineEditForm = OrderDetails.selectAddPOLine();
```

Comment:
No assertion from step 4
"Renewal note" field in "Ongoing order information" accordion contains prefilled text from template

Reply:
Fixed